src/DebugEngineHost.Stub/DebugEngineHost.ref.cs

```diff
@@ -239,14 +266,54 @@ public void Close()
         {
             throw new NotImplementedException();
         }
+    }
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public sealed class HostLogger
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="callback"></param>
+        public static void InitalizeNatvisLogger(Action<LogLevel, string> callback)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
nit: this doesn't need to be in DebugEngineHost.ref.cs since it should only be called by OpenDebugAD7